modules/nasmediaAdmixerBidAdapter.js

```diff
@@ -1,34 +1,37 @@
 import * as utils from '../src/utils';
 import {registerBidder} from '../src/adapters/bidderFactory';
-import find from 'core-js/library/fn/array/find';
+import {BANNER} from '../src/mediaTypes';
+
+const ADMIXER_ENDPOINT = 'https://adn.admixer.co.kr:10443/prebid/ad_req';
+const BIDDER_CODE = 'nasmediaAdmixer';
 
-const ADMIXER_ENDPOINT = 'https://adn.admixer.co.kr:10443/prebid';
 const DEFAULT_BID_TTL = 360;
 const DEFAULT_CURRENCY = 'USD';
 const DEFAULT_REVENUE = false;
 
 export const spec = {
-  code: 'nasmediaAdmixer',
+  code: BIDDER_CODE,
+  supportedMediaTypes: [BANNER],
 
   isBidRequestValid: function (bid) {
-    return !!(bid && bid.params && bid.params.ax_key);
+    return !!(bid && bid.params && bid.params.media_key && bid.params.adunit_id);
   },
 
   buildRequests: function (validBidRequests) {
     return validBidRequests.map(bid => {
-      let adSize = getSize(bid.sizes);
+      const payload = {
+        media_key: utils.getBidIdParameter('media_key', bid.params),
```

Comment:
Is this check needed or can `bid.params.media_key` be accessed directly here? `isBidRequestValid ` should filter out any request missing this param.